STYLEGUIDE.md

```diff
@@ -1,30 +1,31 @@
 # Documentation Style Guide
 
 ### Admin, admin
-Use proper case for the Kong entity, lowercase for the RBAC Role
+Use proper case for the Kong entity, lowercase for the RBAC Role.
 
-* Invite the Admin using the Organization tab
-* The default Roles are `super-admin`, `admin`, and `read-only`
+* Invite the **_Admin_** using the **Organization** tab
+* The default **Roles** are **super-admin**, **_admin_**, and **read-only**
 
 ### Admin API 
-Use proper case
+Use proper case.
 
 ### Admin GUI
-Use *Kong Manager* instead
+Use *Kong Manager* instead.
 
 ### American English
-Use American English throughout Kong documentation
+Use American English throughout Kong documentation.
 
 ### ampersands
-Always use “and” in content, only use “&” in titles with two items that have 
-always had it or in product names
+Always use "and" in content, only use "&" in titles with two items that have 
+always had it or in product names.
 
 ### API
-All upper case; if referring to an entity, use Services and Routes instead
+All upper case; if referring to an entity, use **Services** and **Routes** 
+instead.
 
 ### Basic Auth, Basic Authentication
-Use proper case, abbreviate if the context permits. Only use lowercase with a 
-hyphen in code/config samples
+Use proper case; abbreviate if the context permits. Only use lowercase (with a 
+hyphen) in code or configuration samples.
 
 * Bad: Options include LDAP Auth and basic auth.
```

Comment:
Make this italics

Reply:
(Not sure what "this" refers to--just _configuration_?)

Reply:
oh sorry, bad github linking - the "Bad" in this example is the only "Bad" not in italics